lib/ansible/modules/cloud/amazon/route53_health_check.py

```diff
@@ -314,9 +314,9 @@ def main():
     # Default port
     if port_in is None:
         if type_in in ['HTTP', 'HTTP_STR_MATCH']:
-            port_in = 80
+            port_in = '80'
```

Comment:
Doesn't HealthCheck expect port to be an int? https://github.com/boto/boto/blob/develop/boto/route53/healthcheck.py#L84
Would it make more sense to int() the port to be compared?